### PR TITLE
Milestone 7: Popup window

### PR DESCRIPTION
**Pull request by amejid**
## Personal portfolio milestone 7: Popup window

In this milestone, We:

- Created the projects section dynamically from the stored array
- Created a pop-up window that shows details about the projects
- Added responsiveness to the pop-up

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -2,15 +2,157 @@ const hamburger = document.querySelector('.btn-nav');
...
+    imageUrl: ['', ''],
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    linkLive: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    imageUrl: ['', ''],
```
Comment (Ismailco) [inline js/script.js]: - [ ] [OPTIONAL] Kindly add images to all the projects in the project section, this will make your popup more clean and complete

**Review by Ismailco — APPROVED** [review]
### Hi @amejid, @Owusu-Desmond,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by amejid** [comment]
Thank you for reviewing and approving our code.

File: js/script.js (hunk 1 above)
Comment (amejid) [reply js/script.js]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

**Pull request merged**